metapool/plate.py

```diff
@@ -265,6 +265,8 @@ def dilute_gDNA(plate_df, threshold=15):
 
     # return a copy of the input data. Do not overwrite the input data by
     # default.
+    plate_df['extracted_gdna_concentration_ng_ul'] = \
+        plate_df['Sample DNA Concentration'].copy()
```

Comment:
It looks like a copy is performed on the next line, so I dont think the copy here is needed

Reply:
This copy is to store the extracted gDNA concentration. This dilute_gDNA() function recalculates gDNA concentration for a dilution plate by dividing the original extracted gDNA concentration by a factor of 10, overwriting the Sample DNA Concentration column values.

Reply:
Sure, but isn't this `copy` redundant with the one on the very next line?

Reply:
tbh, I think the df copy of plate_df is a vestige of me working on the functions in jupyter and then moving it into the metapool package. If I remember correctly, the function was modifying the input dataframe (my assumptions of function variables vs global variables(?) were not holding) so I made a copy (df) to make operations on. I get rid of the df = plate_df.copy()

Reply:
nope, never mind. I think this is needed as is. The returned plate_df is a concatenation of subsets of both df and plate_df. I think this function is good as is.